modules/test/playwright/tests/client-extension-web/env/set_up.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+CURRENT_DIR_NAME=$(dirname ${BASH_SOURCE[0]})
+
+echo CURRENT_DIR_NAME=${CURRENT_DIR_NAME}
+
+source ${CURRENT_DIR_NAME}/../../../env/common.sh
+
+main "${@}"
```

Comment:
Already runs `main` in line 20.

Reply:
@kenjiheigel  That's intentional, I am running the main function from the common file. That one declares the variable `_PORTAL_PROJECT_DIR` so I can use in line 15

Reply:
I could do like this, but I wanted to keep the pattern of having a main function like in other setup files

```
#!/bin/bash

CURRENT_DIR_NAME=$(dirname ${BASH_SOURCE[0]})

echo CURRENT_DIR_NAME=${CURRENT_DIR_NAME}

source ${CURRENT_DIR_NAME}/../../../env/common.sh

main "${@}"

combine_properties_files \
  $(get_tomcat_portal_ext_properties_file) \
  \
  ${_PORTAL_PROJECT_DIR}/workspaces/liferay-sample-workspace/configs/local/portal-ext.properties

default_set_up
```